src/dataset.py

```diff
@@ -134,7 +134,7 @@ def load_raw_data(self, data_path) -> Tuple[List,List,List,List,List]:
 
         # BUG: part of the double tokenizing preprocessing error
         with open(data_path+'/sentence.txt', encoding='utf-8') as f:  # only needs tokens as strings
-            sentence = [line for line in f.readlines()]
+            sentence = [line.strip() for line in f.readlines()]
```

Comment:
Not sure if this is really needed, but closer matches what was done debuggin sanity check in `preprocess.ipynb`